datafusion/proto-common/Cargo.toml

```diff
@@ -26,7 +26,7 @@ homepage = { workspace = true }
 repository = { workspace = true }
 license = { workspace = true }
 authors = { workspace = true }
-rust-version = "1.75"
+rust-version = "1.76"
```

Comment:
By my reading of the MSRV policy

https://github.com/apache/datafusion/blob/d8bcff5db22f47e5da778b8012bca9e16df35540/README.md#L100-L103

`1.75` was released [`2023-12-28`](https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1750-2023-12-28) meaning we need to keep the MSRV at 1.75 until 2024-06-28 (6 days from now)

However, since we aren't going to make a release until around July 11 https://github.com/apache/datafusion/issues/11077 this is probably ok